src/Explorer/Controls/Settings/SettingsComponent.tsx

```diff
@@ -0,0 +1,2588 @@
+      </>
+    );
+  };
+
+  private getAnalyticalStorageTtlComponent = (): JSX.Element => {
+    return (
+      <>
+        <div className="formTitle">Analytical Storage Time to Live</div>
+        <div className="tabs disableFocusDefaults" aria-label="Analytical Storage Time to Live" role="radiogroup">
+          <div className="tab">
+            <label tabIndex={0} role="radio" className="disabledRadio">
+              Off
+            </label>
+          </div>
+          <div className="tab">
+            <AccessibleElement
+              as="label"
+              aria-label="AnalyticalStorageTtlOnNoDefaultLabel"
+              tabIndex={0}
+              role="radio"
+              aria-checked={this.state.analyticalStorageTtlSelection.current === TtlType.OnNoDefault ? "true" : "false"}
+              className={`${isDirty(this.state.analyticalStorageTtlSelection) ? "dirty" : ""} ${
+                this.state.analyticalStorageTtlSelection.current === TtlType.OnNoDefault
+                  ? "selectedRadio"
+                  : "unselectedRadio"
+              }`}
+              onActivated={this.onAnalyticalStorageTtlOnNoDefaultKeyPress}
+            >
+              On (no default)
+            </AccessibleElement>
+            <input
+              type="radio"
+              name="analyticalStorageTtl"
+              value={TtlType.OnNoDefault}
+              className="radio"
+              id="analyticalStorageTtlOnNoDefaultId"
+              onChange={this.onAnalyticalStorageTtlSelectionChange}
+              checked={this.state.analyticalStorageTtlSelection.current === TtlType.OnNoDefault}
+            />
+          </div>
+
+          <div className="tab">
+            <AccessibleElement
+              as="label"
+              aria-label="AnalyticalStorageTtlOnLabel"
+              tabIndex={0}
+              role="radio"
+              aria-checked={this.state.analyticalStorageTtlSelection.current === TtlType.On ? "true" : "false"}
+              className={`${isDirty(this.state.analyticalStorageTtlSelection) ? "dirty" : ""} ${
+                this.state.analyticalStorageTtlSelection.current === TtlType.On ? "selectedRadio" : "unselectedRadio"
+              }`}
+              onActivated={this.onAnalyticalStorageTtlOnKeyPress}
+            >
+              On
+            </AccessibleElement>
+            <input
+              type="radio"
+              name="analyticalStorageTtl"
+              value={TtlType.On}
+              className="radio"
+              id="analyticalStorageTtlOnId"
+              onChange={this.onAnalyticalStorageTtlSelectionChange}
+              checked={this.state.analyticalStorageTtlSelection.current === TtlType.On}
+            />
+          </div>
+        </div>
+        {this.state.analyticalStorageTtlSelection.current === TtlType.On && (
+          <>
+            <input
+              type="number"
+              required
+              min="1"
+              max="2147483647"
```

Comment:
This number seems to be already be defined in `EntityPropertyValidationCommons`.